designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ExpressionSuggesterSpec.scala

```diff
@@ -766,6 +770,12 @@ class ExpressionSuggesterSpec
     )
   }
 
+  test("should suggest methods for unknown") {
+    spelSuggestionsFor("#unknown.") shouldBe List(
+      suggestion("canCastTo", Typed[Boolean]),
```

Comment:
Under default settings (with no dynamic access on default), this is misleading. As I understand it, by default we allow casting only on Maps and Lists? If we are to add this suggestions on Unknown, this makes some problems, like suggestions on things like `unknown.` without `#` or `#UTIL.canCastTo()` I think we should not have any suggestions on Unknown.

Reply:
I cannot agree with you on this. I think `toString` and casting methods should be available on the unknown and invoking other methods should be limited by a property: `methodExecutionForUnknownAllowed` because we cannot be sure whether other methods are available for unknown. Without this behaviour `casting` methods are rather unusable.

Reply:
Ok, I agree casting will be kind of useless when it would be working only on Lists/Arrays and Maps. Im not opposed to adding casting on Unknown and having these suggestions, there may be some edge cases to cover